atqa-framework/tests/pages/ExtensionInitPage.ts

```diff
@@ -0,0 +1,23 @@
+import { BasePage } from './BasePage';
+
+export class ExtensionInitPage extends BasePage {
+  public SELECTORS = {
```

Comment:
There is no need to explicitly specify the `public` data modifier for class attributes/methods, it is used by default

Reply:
fixed